packages/util/lib/zeus-api.js

```diff
@@ -10,3 +10,27 @@ export const triggerKeyValuePairsUpdate = (kvps = {}) => {
     window.zeus.emit("RESET_KEYVALUES", kvps);
   }
 };
+
+export const forceRefresh = slotIds => {
+  if (Array.isArray(slotIds)) {
```

Comment:
MINOR: This duplication could probably be prevented. You could probably have something like:

```js
export const forceRefresh = slotIds => {
  const renderFunc = slotId => {
    const element = document.getElementById(getSlotId(slotId));
    element && !!element.render && element.render();
  };

  if (Array.isArray(slotIds)) {
    slotIds.forEach( renderFunc );
    return;
  }
  
  renderFunc(slotIds);
  return;
}
```